includes/validation/class-amp-validation-manager.php

```diff
@@ -368,6 +368,44 @@ public static function has_cap() {
 		return current_user_can( 'edit_posts' );
 	}
 
+	/**
+	 * Determine whether a validation error should be sanitized.
+	 *
+	 * @param array $error Validation error.
+	 * @return bool Whether error should be sanitized.
+	 */
+	public static function is_validation_error_sanitized( $error ) {
+		$term_data = AMP_Validation_Error_Taxonomy::prepare_validation_error_taxonomy_term( $error );
+
+		$term = get_term_by( 'slug', $term_data['slug'], AMP_Validation_Error_Taxonomy::TAXONOMY_SLUG );
+		if ( isset( self::$validation_error_status_overrides[ $term_data['slug'] ] ) ) {
+			$sanitized = AMP_Validation_Error_Taxonomy::VALIDATION_ERROR_ACCEPTED_STATUS === self::$validation_error_status_overrides[ $term_data['slug'] ];
+		} elseif ( ! empty( $term ) && AMP_Validation_Error_Taxonomy::VALIDATION_ERROR_ACCEPTED_STATUS === $term->term_group ) {
```

Comment:
This could be simplified to:
``` PHP
// ...
else {
	$sanitized = ! empty( $term ) && AMP_Validation_Error_Taxonomy::VALIDATION_ERROR_ACCEPTED_STATUS === $term->term_group;
}
```